man/systemd.network.xml

```diff
@@ -205,6 +205,16 @@
           controlled by other applications.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Group=</varname></term>
+        <listitem>
+          <para> Link groups are similar to port ranges found in managed switches.
```

Comment:
Don't know whether it matters, but there is an extraneous white space after the opening tag.